spirv_parser.cpp

```diff
@@ -975,9 +978,16 @@ void Parser::parse(const Instruction &instruction)
 		current_block->condition = ops[0];
 		current_block->default_block = ops[1];
 
+		uint32_t remaining_ops = length - 2;
 		for (uint32_t i = 2; i + 2 <= length; i += 2)
 			current_block->cases.push_back({ ops[i], ops[i + 1] });
 
+		if (remaining_ops % 3)
```

Comment:
Shouldn't this be remaining_ops % 3 == 0?

Reply:
Yes, it should. I'm sorry for this.